packages/bigtable/README.md

```diff
@@ -0,0 +1,109 @@
+# @google-cloud/bigtable
+> Google Cloud Bigtable Client Library for Node.js
+
+*Looking for more Google APIs than just Bigtable? You might want to check out [`google-cloud`][google-cloud].*
+
+- [API Documentation][gcloud-bigtable-docs]
+- [Official Documentation][cloud-bigtable-docs]
+
+*You may need to [create a cluster][cloud-bigtable-cluster] to use the Google Cloud Bigtable API with your project.*
+
+
+```sh
+$ npm install --save @google-cloud/bigtable
+```
+```js
+var bigtable = require('@google-cloud/bigtable')({
+  projectId: 'grape-spaceship-123',
+  keyFilename: '/path/to/keyfile.json',
+  zone: 'my-zone',
+  cluster: 'my-cluster'
+});
+
+var table = bigtable.table('prezzy');
+
+table.getRows(function(err, rows) {});
```

Comment:
Should we expand on this with a comment or something? e.g. *Error handling omitted. `rows` is an array of Row objects.*?